src/components/ConnectWallet/ConnectWalletModal.tsx

```diff
@@ -109,16 +131,19 @@ export const DisconnectWalletModal: FC<DisconnectWalletModalProps> = ({
 
   return (
     <ConnectWalletModalBase isOpen={isOpen} onClose={onClose}>
-      <div className="flex h-full items-start justify-end">
+      <div className="grid grid-cols-6 grid-rows-1 gap-4">
+        <div></div>
```

Comment:
Is this to create an empty column?

You can use `col-start-2` on the `Dialog.Title` element if you want to skip a column.